contrib/face_recognition.lua

```diff
@@ -92,8 +92,10 @@ local function build_image_table(images)
   end
 
   for _,img in ipairs(images) do
-    image_table[img] = tmp_dir .. df.get_basename(img.filename) .. file_extension
-    cnt = cnt + 1
+    if img ~= nil then
```

Comment:
Has you ever found an image that didn't exist?  Since images is dt.gui.action.images, this would be a bug in darktable, passing a non-existent image as part of the arguments.

Reply:
I ran face_recognition.lua countless times in the last two weeks in order to realize how it was failing. Random (and very rare) times it failed trying to access image_table[img]. I cant tell why because most of times I was just running the same data set all over again without any error in this line. I decided doing a sanity check because I couldn’t reproduce the error in a deterministic way to raise an issue in darktable project.

Reply:
I saw where you did another check later and wondered if that was the case.